docs/base-maps.md

```diff
@@ -0,0 +1,29 @@
+# Base Maps
+
+While deck.gl works independently without any map component, when visualizing geospatial datasets, a base map can offer invaluable context for understanding the overlay layers.
+
+## Base Maps Renderers
+
+deck.gl has been designed to work in tandem with popular JavaScript base map providers. Depending on your tech stack, deck.gl's support for a particular base map solution may come with different level of compatibility and limitations.
+
+There are two types of integration between deck.gl and a base map renderer:
+
+- **Overlaid**: the Deck canvas is rendered over the base map as a separate DOM element. Deck's camera and the camera of the base map are synchronized so they pan/zoom together. This is the more robust option since the two libraries manage their renderings independently from each other. It is usually sufficient if the base map is 2D.
+
+![Deck as overlay on top of the base map](https://miro.medium.com/max/1600/0*K3DVssEhnv5VaDCp)
+
+- **Interleaved**: Deck renders into the WebGL context of the base map. This allows for occlusion between deck.gl layers and the base map's labels and/or 3D features. The availability of this option depends on whether the base map solution exposes certain developer APIs, and may subject the user to bugs/limitations associated with such APIs.
+
+![Deck interleaved with base map layers](https://miro.medium.com/max/1600/0*faYL1UbVD4af5qzy)
+
+
+| Library | Pure JS | React | Overlaid | Interleaved |
```

Comment:
Since this document is effectively the "introduction" to base map integration, we should add a column to this table that links to the respective pages in developer-guide/base-maps

Reply:
done